modules/installation-creating-aws-dns.adoc

```diff
@@ -0,0 +1,136 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_aws_upi/installing-aws-upi.adoc
+
+[id="installation-creating-aws-dns-{context}"]
+= Creating networking and load balancing components in AWS
+
+You must configure networking and load balancing in Amazon Web Services (AWS) for your
+{product-title} cluster to use. The easiest way to create these components is
+to modify the provided CloudFormation template, which also creates a hosted zone
+and subnet tags.
+
+You can run the template multiple times within a single VPC.
+
+[NOTE]
+====
+If you do not use the provided CloudFormation template to create your AWS
+infrastructure, you must review the provided information and manually create
+the infrastructure. If your cluster does not initialize correctly, you might
+have to contact Red Hat support with your installation logs.
+====
+
+.Prerequisites
+
+* Configure an AWS account.
+* Generate the Ignition config files for your cluster.
+* Create and configure a VPC and assocated subnets in AWS.
+
+.Procedure
+
+. Create a YAML file that contains a copy of the CloudFormation template that
+describes the networking objects and load balancers and save it to your
+computer.
+
+. Create a JSON file that contains the parameter values that the template
+requires:
++
+[source,json]
+----
+[
+  {
+    "ParameterKey": "ClusterName", <1>
```

Comment:
What is the difference between ClusterName and InfrastructureName?  Are these two values different than `jq -r .infraID /some/dir/openshift-abcxyz/metadata.json`?